src/promptflow-devkit/promptflow/_sdk/_orchestrator/run_submitter.py

```diff
@@ -153,7 +153,7 @@ def _submit_bulk_run(
         )
         try:
             batch_engine = BatchEngine(
-                flow.path,
+                run._dynamic_callable or flow.path,
```

Comment:
so dynamic callable will drop the path?
all features related to paths will be out of usage?

Reply:
it won't. it will use the class address as entry and generate a temp `flow.flex.yaml` for path related features.